nautobot_golden_config/views.py

```diff
@@ -585,3 +586,16 @@ def post(self, request):
             **job.job_class.serialize_data(request),
         )
         return redirect(job_result.get_absolute_url())
+
+
+class GenerateIntendedConfigView(PermissionRequiredMixin, TemplateView):
+    """View to generate the intended configuration."""
+
+    template_name = "nautobot_golden_config/generate_intended_config.html"
+    permission_required = ["dcim.view_device", "extras.view_gitrepository"]
```

Comment:
```suggestion
    permission_required = ["dcim.view_device", "extras.view_gitrepository", "nautobot_golden_config.view_goldenconfig"]
```

Reply:
I could see a user wanting to require a permission to use this but I don't think `view_goldenconfig` is the correct permission. Perhaps we need to implement a new permission similar to `run_job`?